include/ensmallen_bits/cmaes/cmaes.hpp

```diff
@@ -153,13 +151,16 @@ class CMAES
 
   //! The selection policy used to calculate the objective.
   SelectionPolicyType selectionPolicy;
+
+  TransformationPolicyType transformationPolicy;
```

Comment:
Want to add a comment here on what this is for?  Maybe

```
//! The transformation policy maps candidate points back into a suitable domain.
```

but I am not sure if that fully reflects your intention, it's just a suggestion.

Reply:
I'll definitely add a comment.